rules/shorthandData.js

```diff
@@ -2,8 +2,34 @@
 
 // See https://github.com/stylelint/stylelint/blob/10.1.0/lib/reference/shorthandData.js
 module.exports = {
-	margin: ['margin-top', 'margin-bottom', 'margin-left', 'margin-right'],
-	padding: ['padding-top', 'padding-bottom', 'padding-left', 'padding-right'],
+	margin: [
+		'margin-top',
+		'margin-bottom',
+		'margin-left',
+		'margin-right',
+		'margin-block',
+		'margin-block-end',
+		'margin-block-start',
+		'margin-inline',
+		'margin-inline-end',
+		'margin-inline-start',
```

Comment:
Is it needed to duplicate properties here, since they have separate declarations below?

Reply:
Yes, will do that. Might not have time for the tweaks until Monday, but I won't forget about it.

Reply:
And I will write tests and find out if they are necessary.